src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int numberBall = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery[] lotteries = new Lottery[numberBall];
+        for (int i = 0; i < lotteries.length; i++) {
+            lotteries[i] = new Lottery();
+            System.out.println(lotteries[i].getRandomBall());
+        }
```

Comment:
Good! But do you actually need a 3 `Lottery` here. I suppose we can cope only with one :)

Let's create one lottery, and getting all 3 random balls from it. Also, you don't need an array here. You can simply use your BALLS_COUNT in loop declaration.